pb/config/config.proto

```diff
@@ -221,6 +216,11 @@ message Vserver {
   // Warning messages about this Vserver (such as misconfigured backends) to be
   // displayed on operator consoles.
   repeated string warning = 9;
+
+  // The list of backends for this vserver.
+  repeated Backend backend = 10;
+
+  reserved 6; // was legacy_backend
```

Comment:
The field name "legacy_backend" should be reserved as well.

Reply:
Done.